ocs_ci/ocs/node.py

```diff
@@ -1686,3 +1688,19 @@ def get_node_mon_ids(node_name):
     mon_pods = pod.get_mon_pods()
     node_mon_pods = get_node_pods(node_name, pods_to_search=mon_pods)
     return [pod.get_mon_pod_id(mon_pod) for mon_pod in node_mon_pods]
+
+
+def add_new_disk_for_vsphere(sc_name):
```

Comment:
is this function used only on vsphere deployments? if so, shouldn't be a check for deployment type? with a error or skip if the type is not vmware?

Reply:
My assumption is that if someone uses this function, he will use it only for vSphere.